app.config.ts

```diff
@@ -66,5 +81,8 @@ export default ({ config }: any): any => {
         },
       ],
     },
+    expo: {
```

Comment:
https://docs.expo.dev/guides/using-sentry/#42-add-the-config-plugin required to define now